CONTRIBUTING.rst

```diff
@@ -247,12 +247,12 @@ Example entry:
     - Fix off-by-one error when setting ticks in :func:`~plotting.plot_surf` (:gh:`3105` by `Dimitri Papadopoulos Orfanos`_).
   
 Coding Style
--------------
+------------
 
 The nilearn codebase follows PEP8_ styling.
 The main conventions we enforce are :
 
-- line length < 80
+- line length < 90
```

Comment:
Shouldn't we keep this as the flake8 default or is that too short?

Reply:
just reverted to use line length 79
made it explicit in all config files